packages/tools/devtools/devtools-browser-extension/webpack.test.js

```diff
@@ -50,6 +50,13 @@ module.exports = (env) => {
 				template: "./e2e-tests/app/index.html",
 			}),
 		],
+		// This impacts which files are watched by the dev server (and likely by webpack if watch is true).
+		// This should be configurable under devServer.static.watch
+		// (see https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md) but that does not seem to work.
+		// The CLI options for disabling watching don't seem to work either, so this may be a symptom of using webpack4 with the newer webpack-cli and webpack-dev-server.
+		watchOptions: {
```

Comment:
I'm curious what the motivation for this is. Don't we want the app to reload if the dependencies change?

Reply:
Also, we aren't using webpack 4, we're using webpack 5, so the comment above doesn't seem right.

Reply:
Removed

Reply:
Re-opening